runtime/src/main/kotlin/kotlin/coroutines/CancellationException.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package kotlin.coroutines.cancellation
+
+/**
+ * Thrown by cancellable suspending functions if the coroutine is cancelled while it is suspended.
+ * It indicates _normal_ cancellation of a coroutine.
+ */
+public open class CancellationException : IllegalStateException {
```

Comment:
Also makes sense to annotate with `@SinceKotlin("1.4")`

Reply:
Agree. Done.